src/nav/dashboards.yml

```diff
@@ -47,6 +47,8 @@ pages:
             path: /docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-how-nrql-works
           - title: NRQL charts and dashboards
             path: /docs/query-your-data/nrql-new-relic-query-language/get-started/charts-and-dashboards-with-nrql 
+          - title: NRQL tutorial - Introduction
```

Comment:
I'm not 100% sure on this title in the nav. There are going to be other tutorials over the next few sprints and we decided not to refer to assign levels to them, so this is the best I could come up with. If anyone can think of anything better I please don't hesitate to let me know!

Reply:
I haven't really decided everything exactly yet. I do plan on getting rid of the Get Started section when we move NRQL to its own section in the home nav to increase visibility, and the tutorials will be under their own section. I think that's going to get done after the second tutorial is finished and I get a few more topics created or revamped via working with PM and Dev. We're kind of in this weird in-between state with topic names in the nav right now because I haven't done enough I think to justify the move yet.